docs/getting-started/12-bind-redis-service-to-function.md

```diff
@@ -0,0 +1,212 @@
+
+     > **TIP:** It is considered good practice to use **envPrefix**. In some cases, a Function must use several instances of a given ServiceClass. Prefixes allow you to distinguish between instances and make sure that one Secret does not overwrite another one.
+
+4. Check that the ServiceBindingUsage CR was created. The last condition in the CR status should be `Ready True`:
+
+  ```bash
+  kubectl get servicebindingusage orders-function -n orders-service -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+  ```
+
+If you want to see the Secret details and retrieve them from the ServiceBinding, run this command:
+
+  ```bash
+  kubectl get secret orders-function -n orders-service -o go-template='{{range $k,$v := .data}}{{printf "%s: " $k}}{{if not $v}}{{$v}}{{else}}{{$v | base64decode}}{{end}}{{"\n"}}{{end}}'
+  ```
+
+  Expect a response similar to this one:
+
+  ```bash
+  HOST: hb-redis-micro-0e965585-9699-443f-b987-38bc6af0e416-redis.serverless.svc.cluster.local
+  PORT: 6379
+  REDIS_PASSWORD: 1tvDcINZvp
+  ```
+
+  </details>
+  <details>
+  <summary label="console-ui">
+  Console UI
+  </summary>
+
+1. Go to **Development** > **Functions** in the left navigation panel and select `orders-function`.
+
+2. Switch to the **Configuration** tab and select **Create Service Binding** in the **Service Bindings** section.
+
+3. Select `redis-service` in the **Service Instance** drop-down list, add `REDIS_` as **Prefix for injected variables**, and make sure **Create new Secret** is selected.
+
+   > **NOTE:** The **Prefix for injected variables** field is optional. It adds a prefix to all environment variables injected in a Secret to the Function when creating a ServiceBinding. In our example, the prefix is set to `REDIS_`, so all environmental variables will follow the `REDIS_{ENVIRONMENT_VARIABLE}` naming pattern.
+
+   > **TIP:** It is considered good practice to use prefixes for environment variables. In some cases, a Function must use several instances of a given ServiceClass. Prefixes allow you to distinguish between instances and make sure t
+
+4. Select **Create** to confirm the changes.
+
+A message will appear on the screen confirming that the ServiceBinding was created, and you will see it in the **Service Bindings** section in your Function along with environment variable names.
+
+If you switch to the **Code** tab and scroll down to the **Environment Variables** section, you should see `REDIS_PORT`, `REDIS_HOST` and `REDIS_REDIS_PASSWORD` items with the `Service Binding` type. It indicates that the environment variable was injected to the Function by the ServiceBinding.
+
+    </details>
+</div>
+
+### Call and test the Function
+
+Follow these steps:
+
+> **CAUTION:** If you have a Minikube cluster, you must first add the IP address of the exposed Service to the `hosts` file on your machine:
```

Comment:
```suggestion
> **CAUTION:** If you have a Minikube cluster, you must first add its IP address to the `hosts` file on your machine:
```

or ```(...) add its IP address mapped to the domain of the exposed Service to the `hosts` file on your machine:```

What's really added to `/etc/hosts` is the Minikube cluster's IP (`minikube ip`) mapped to the service's hostname/domain (`orders-function.kyma.local`) so that accessing the domain can redirect you to the cluster.